bundles/org.openhab.binding.tr064/README.md

```diff
@@ -35,48 +37,97 @@ For security reasons it is highly recommended to set both, username and password
 Values start with `0`, including the number of "Call Blocks" (two configured call-blocks -> first deflection is `2`).
-This is an optional parameter and multiple values are allowed.
+This is an optional parameter and multiple values are allowed:  add one value per line in the Main User Interface.
 
 Most devices support call lists.
-The binding can analyze these call lists and provide channels for the number of missed calls, inbound calls, outbound calls and rejected (blocked) calls.
-The days for which this analysis takes place can be controlled with the `missedCallDays`, `rejectedCallDays`, `inboundCallDays`, `outboundCallDays` and `callListDays`.
-This is an optional parameter and multiple values are allowed.
+The binding can retrieve these call lists and provide channels for the number of missed calls, inbound calls, outbound calls and rejected (blocked) calls,
+for a given number of days. A channel is added to the device if such a number is set through the corresponding parameter
+in the Main User Interface. The parameters are: `missedCallDays`, `rejectedCallDays`, `inboundCallDays`, `outboundCallDays` and `callListDays`.
 
 Since FritzOS! 7.20 WAN access of local devices can be controlled by their IPs.
 If the `wanBlockIPs` parameter is set, a channel for each IP is created to block/unblock WAN access for this IP.
 Values need to be IPv4 addresses in the format `a.b.c.d`.
-This is an optional parameter and multiple values are allowed.
+This is an optional parameter and multiple values are allowed:  add one value per line in the Main User Interface.
 
 If the `PHONEBOOK` profile shall be used, it is necessary to retrieve the phonebooks from the FritzBox.
-The `phonebookInterval` is uses to set the refresh cycle for phonebooks.
+The `phonebookInterval` is uses to set the refresh cycle for phonebooks. It defaults to 600 seconds,
+and it can be set to 0 if phoneooks are not to be used.
+
 
 ### `subdevice`, `subdeviceLan`
 
-Besides the bridge that the thing is attached to, sub-devices have a `uuid` parameter.
-This is the UUID/UDN of the device and a mandatory parameter.
-Since the value can only be determined by examining the SCPD of the root device, the simplest way to get hold of them is through auto-discovery.
+Additional informations (i.e. channels) are available in subdevices of the rootdevice, which is therefore a bridge.
+Each subdevice is characterized by a unique `uuid` parameter: this is the UUID/UDN of the device. 
+This is a mandatory parameter to be set in order to add the subdevice. Since the parameter value can only be determined
+by examining the SCPD of the root device, the simplest way to obtain it is through auto-discovery.
+
+For FritzBox 7590 and for FritzBox 7360v1 (but likely many others) three subdevices are available (and found through auto-discovery):
+a LAN Device (of type `subdeviceLan`), a WAN Device (of type `subdevice`) and a WANConnection Device (of type `subdevice`).
```

Comment:
I would suggest to remove this device-specific comment. Discovery shows all sub devices.

Reply:
I simplified that sentence.